tests/test_db.py

```diff
@@ -412,19 +412,13 @@ def test_is_old_schema(tmp_path):
     responses.add(responses.GET, url_version, status=200, body="1")
     responses.add(responses.GET, url_version, status=200, body="42")
 
-    # When the remote version file doesn't exist, we consider the db as being old.
-    assert db.is_old_schema(db_path)
+    # When the remote version file doesn't exist, we consider the db as being new.
+    assert db.is_different_schema(db_path)
 
-    # When the remote version file doesn't exist, we consider the db as being old.
-    assert db.is_old_schema(db_path)
-
-    # When the remote version file exists and returns the same version as the current db, we consider the remote db as not being old.
-    assert not db.is_old_schema(db_path)
-
-    # When the remote version file exists and returns a newer version than the current db, we consider the remote db as not being old.
-    assert not db.is_old_schema(db_path)
+    # When the remote version file exists and returns a newer version than the current db, we consider the remote db as not being old.(different version)
+    assert not db.is_different_schema(db_path)
 
     db.register(db_path, url_zst, 43, support_files=[])
 
-    # When the remote version file exists and returns an older version than the current db, we consider the remote db as being old.
-    assert db.is_old_schema(db_path)
```

Comment:
You should also add a test with a version that is older than the one returned by requests (you can call db.register with 21 and then assert db.is_different_schema)

Reply:
@webbdays don't forget about this

Reply:
That is already included

Reply:
4th assert

Reply:
Good, I didn't notice.